data/remnant missions.txt

```diff
@@ -495,3 +560,713 @@ mission "Remnant: Technology Available"
+			`	After you finish your conversation with her, you stroll off to look around the spaceport, enjoying the fact that you can now understand a bit of the conversations going on around you. It is only the most basic elements, but it is a good start. You realize, however, that you have a lot of practice to do before you become fluent.`
+
+event "remnant: sign studies complete"
+
+
+
+mission "Remnant: Learn Sign 3"
+	name "Conversational in Remnant Sign"
+	description `Return to <destination> to see how well you can sign.`
+	destination "Caelian"
+	invisible
+	to offer
+		has "event: remnant: sign studies complete"
+	on complete
+		log "Have achieved a degree of fluency in the Remnant sign language sufficient for general conversation."
+		dialog
+			`As you make your way into the Caelian spaceport, it occurs to you that the information director who gave you the first instructional videos might appreciate how much you have learned over the past few weeks of practice. You stretch your fingers and stop by the desk, where you are able to have a simple conversation with the information director.`
+			`	"Congratulations, Captain <last>!" she signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways that just by blood. I look forward to seeing what results from your work with us." She is interrupted by the arrival of other Remnant asking her something, and she bids you farewell with an apologetic gesture.`
+
+
+
+mission "Remnant: Salvage 1"
+	name "Remnant History Updates"
+	description `Find a copy of human history on <destination> for the Remnant.`
+	minor
+	source "Viminal"
+	destination "Alexandria"
+	to offer
+		has "Remnant: Learn Sign 2: done"
+		random < 50
+		or
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Plasma Cannons: done"
+			has "Remnant: Heavy Laser: done"
+	on offer
+		conversation
+			`As you step out into the icy chill of Viminal, you get a message from Tali, the engineering prefect, asking if you could meet her in the cafeteria.`
+			choice
+				`	(Not right now.)`
+					defer
+				`	(Accept her invitation.)`
+			`	Replying that you can, you make your way through the crowds and find an empty table. Moments later Tali appears and grabs two trays from the dispenser before heading towards you. She settles down in the opposite chair with a look of exhaustion.`
+			`	"It is good to see you again, <first>," she sings softly. When you sign a reply she sits up and smiles. "My, you are a quick learner," she signs. "Great, this is much faster."`
```

Comment:
Consider removing the `"My, . . ."` here, if you are talking to someone who barely knows your language you would try to avoid excess words. You could argue that it is part of her personality but then you should probably add something about her being hard to follow.

Reply:
OK, I replaced that line with:

`	"It is good to see you again, <first>," she sings softly. When you sign a reply she sits up, smiles, and switches to sign. "My, you are a quick learner. This is much faster." Her quick gestures are hard to follow, but the language videos seem to have focused on a practical vocabulary that has most of the words she uses.`

Does that work better?